gripper_controller:
  ros__parameters:
    open_angle: 38     # degrees
    close_angle: 3     # degrees
    gpio_chip: "/dev/gpiochip1"
    gpio_line: 14
    period: 0.02       # seconds
    min_us: 500        # microseconds for min_angle
    max_us: 2500       # microseconds for max_angle
    min_angle: 0       # degrees
    max_angle: 270     # degrees
